examples/seismic/model.py

```diff
@@ -503,3 +507,108 @@ def vp(self, vp):
             initialize_function(self.m, 1 / (self.vp * self.vp), self.nbpml)
         else:
             self.m.data = 1 / vp**2
+
+
+class ModelElastic(object):
```

Comment:
I'm sure this should inherit from `Model` and some things (attributes, methods, etc) should/could be reused

Reply:
done